lib/services.rb

```diff
@@ -2,8 +2,4 @@ module Services
   def self.content_store
     @content_store ||= GdsApi::ContentStore.new(Plek.new.find("content-store"))
   end
-
-  def self.rate_limiter
-    @rate_limiter ||= Ratelimit.new("email-alert-api:deliveries")
```

Comment:
Note to self: double-checked this prefix matches [the automatic namespace from govuk_sidekiq](https://github.com/alphagov/govuk_sidekiq/blob/master/lib/govuk_sidekiq/railtie.rb#L16).

Reply:
This is the namespace it comes out with:

```
irb(main):069:0> Redis.new.exists("email-alert-api:ratelimit:send_email:requests")
=> true
```

Note the key only lasts as long as the rate limit is running so you'll find in quiet times that this returns false.

Reply:
Thanks for clarifying 👍